scripts/secnetperf-helpers.psm1

```diff
@@ -131,11 +138,21 @@ function Install-XDP {
     $installerUri = (Get-Content (Join-Path $PSScriptRoot "xdp.json") | ConvertFrom-Json).installer
     $msiPath = Repo-Path "artifacts/xdp.msi"
     Write-Host "Downloading XDP installer"
-    Invoke-WebRequest -Uri $installerUri -OutFile $msiPath
+    whoami
+    Invoke-WebRequest -Uri $installerUri -OutFile $msiPath -UseBasicParsing
```

Comment:
What's the `-UseBasicParsing` do?

Reply:
This option is necessary on older versions of powershell. Once Azure implements support for managed images on the exp. boost sku, we can assume all environments will have powershell 7 installed. For now, this is to avoid having to install pwsh 7 each run (fails from time to time). Let's chat about this.